images/win/scripts/ImageHelpers/PathHelpers.ps1

```diff
@@ -1,68 +1,56 @@
-function Test-MachinePath{
+function Get-SystemVariable{
```

Comment:
Let's use the same syntax style for consistence:

```
function Verb-Noun {
}
```

Reply:
Updated